pkg/device/device.go

```diff
@@ -0,0 +1,41 @@
+package device
```

Comment:
further, the C macro only does bit-math. Something eventually easier to just do natively, rather than importing from C...